app/models/subcase-type.js

```diff
@@ -1,12 +1,10 @@
 import Model, { hasMany, attr } from '@ember-data/model';
 
-// TODO: octane-refactor
-// eslint-disable-next-line ember/no-classic-classes
-export default Model.extend({
-  label: attr('string'),
-  altLabel: attr('string'),
-  scopeNote: attr('string'),
-  subcases: hasMany('subcase', {
-    inverse: null,
-  }),
-});
+export default class Subcase extends Model {
```

Comment:
Model name is incorrect

Reply:
Darn, I need to pay more attention :facepalm: